packages/transactions/tests/builder.test.ts

```diff
@@ -921,7 +1014,7 @@ test('make a multi-sig contract call', async () => {
   );
 });
 
-test('Estimate transaction transfer fee', async () => {
+test.skip('Estimate transaction transfer fee', async () => {
```

Comment:
The `.skip` tests were previously not run and are now outdated/fail. https://github.com/hirosystems/stacks.js/issues/1175 addresses fixing them in a separate PR.